presto-accumulo/src/main/java/com/facebook/presto/accumulo/index/IndexLookup.java

```diff
@@ -362,57 +373,63 @@ private long getNumRowsInTable(String metricsTable)
     private List<Range> getIndexRanges(String indexTable,
             Multimap<AccumuloColumnConstraint, Range> constraintRanges,
             Collection<Range> rowIDRanges)
-            throws TableNotFoundException
+            throws TableNotFoundException, ExecutionException, InterruptedException
     {
-        Set<Range> finalRanges = null;
-        // For each column/constraint pair
+        final Set<Range> finalRanges = new HashSet<>();
```

Comment:
Does this need to be final?  Presto style doesn't accept final local member variables.